site/docs/resources/faq.md

```diff
@@ -115,6 +115,12 @@ There are a few things you can do.
 4. Come ask us in the group chat for help. We have experienced people there.
 5. It is possible to ask Telegram to increase the limits, but this is very unlikely to happen if you did not do steps 1-3 first.
 
+### Cannot find type definition file for 'node-fetch'.
+
+This is the result of a version conflict between the installed `@types/node-fetch`, and the grammY dependency `node-fetch`.
```

Comment:
It's actually just missing type information. There's no mismatch. We don't have the types at a production dependency, so they're missing if users try to type-check their node modules.

Skipping this step of checking everything is the best solution.

Alternatively, users can install the package `@types/node-fetch@2`.

Reply:
I modified it.